tick/optim/solver/svrg.py

```diff
@@ -51,14 +57,25 @@ class SVRG(SolverFirstOrderSto):
         Information along iteration is recorded in history each time the
         iteration number of a multiple of ``record_every``
 
```

Comment:
Maybe `variance_reduction` and `delayed_updates` are technical enough to go in an Other Parameters section?

Reply:
Yes indeed, I've moved them